core/domain/exp_services_test.py

```diff
@@ -6513,34 +6566,25 @@ def setUp(self):
         self.draft_change_list_dict = [
             change.to_dict() for change in self.draft_change_list]
         # Explorations with draft set.
-        user_models.ExplorationUserDataModel(
-            id='%s.%s' % (self.USER_ID, self.EXP_ID1), user_id=self.USER_ID,
-            exploration_id=self.EXP_ID1,
-            draft_change_list=self.draft_change_list_dict,
-            draft_change_list_last_updated=self.DATETIME,
-            draft_change_list_exp_version=1,
-            draft_change_list_id=2).put()
-
-        migration_change_list = [exp_domain.ExplorationChange({
-            'cmd': exp_domain.CMD_MIGRATE_STATES_SCHEMA_TO_LATEST_VERSION,
-            'from_version': '0',
-            'to_version': '1'
-        })]
-        exp_services._save_exploration(  # pylint: disable=protected-access
-            self.USER_ID, exploration, 'Migrate state schema.',
-            migration_change_list)
+        exp_user_data = user_models.ExplorationUserDataModel.create(
+            self.USER_ID, self.EXP_ID1)
+        exp_user_data.draft_change_list = self.draft_change_list_dict
+        exp_user_data.drafdraft_change_list_exp_versiont_change_list_last_updated = (  # pylint: disable=line-too-long
```

Comment:
Isn't this wrong?

Reply:
Actually, I have just changed the order of `migration_change_list` and the draft because when we use `update_exploration` instead of the direct `_save_exploration` function then the exploration gets updated according to the draft which we do not want because we want to test that the draft is applied after the exploration is upgraded.

Code for `migration_change_list` is moved to lines: 6546-6554 (just above this draft setting code)

PS: If instead of this, you are referring to some other part then please specify.

Reply:
I mean especially the property name `drafdraft_change_list_exp_versiont_change_list_last_updated`

Reply:
Ohh so sorry! I missed this by mistake. I have fixed this property.
Tysm.